www/netj00/HW2/index.css

```diff
@@ -0,0 +1,100 @@
+.fontNotSerif{
+    font-family: Arial, Helvetica, sans-serif;
+}
+.redFont{
+    color:red;
+}
+.greenFontRGB{
+    color: #008000;
+}
+.boltBlueHex{
+    font-weight: bold;
+    color: #2e39ff;
+}
+.size16{
+    font-size: 18px;
+}
+.italic24px{
+    font-style: italic;
+    font-size: 24px;
+}
+.whiteFontWithBrownBackground{
+    color: white;
+    background-color: saddlebrown;
+}
+.centerLineHeight30{
+    text-align: center;
+    line-height: 32px;
+}
+.justifyCAPITAL{
+    text-align: justify;
+    text-transform: uppercase;
+}
+.box400x300PurpleBorder1px30padding{
+    width: 400px;
+    height: 300px;
+    border: 1px solid purple;
+    padding: 30px;
+}
+.box2{
+    width: 500px;
+    height: 200px;
+    background-color: #ff2025;
+    margin:100px;
+}
+.hidden{
+    visibility: hidden;
+}
+.box3{
+    width: 100%;
+    height: 150px;
+    background-color: blue;
+    padding: 50px;
+    border: 10px solid yellow;
+}
+.gap{
+    color: white;
+    background-color: white;
+}
+.box4{
+    width: 200px;
+    height: 150px;
+    color: rgba(51, 255, 14, 0.5);
+}
+.parent{
+    position: relative;
+}
+.box5{
+    width: 400px;
+    height: 300px;
+    background-color: lightblue;
+    position: relative;
+}
+.box6{
+    width: 50px;
+    height: 50px;
+    background-color: pink;
+    position: absolute;
+    right: 10px;
+    top: 50px;
+}
+.button1{
+    width: 300px;
+    height: 100px;
+}
+.button1:focus{
+    width: 300px;
```

Comment:
rozdil mezi :focus a :active

:active = kdyz se na element klikne nebo se vybere
:focus = posledni :active

takze kdyz kliknes na button, on se stava :active i :focus, :active zustane pouze po dobu kliku ale :focus zustane dokud nekliknes na jiny element, v uloze jsem chtel :active

navic se ma menit barva textu a ne backgroundu, a jeste k tomu to ma byt modra barva